pkgs/os-specific/linux/wooting-xinput-udev-rules/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "wooting-xinput-udev-rules";
+  version = "20201129";
+
+  # Source: https://wooting.helpscoutdocs.com/article/83-guide-configuring-xinput-support-for-the-wootings-under-linux
+  src = [ ./wooting-xinput.rules ];
+
+  unpackPhase = ":";
+
+  installPhase = ''
+    install -Dpm644 $src $out/lib/udev/rules.d/71-wooting.rules
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://wooting.helpscoutdocs.com/article/83-guide-configuring-xinput-support-for-the-wootings-under-linux";
+    description = ''
+      udev rules that give NixOS permission to communicate with Wooting keyboards in xinput mode
+    '';
+    platforms = platforms.linux;
+    license = "unknown";
```

Comment:
if there's no license, I think we have to assume that it's unfree.

Reply:
Fixed!